### PR TITLE
[RM Chap04] Modifications of table 4-24

### PR DESCRIPTION
**Pull request by karinesevilla**
Resolve Issue#385 https://github.com/cntt-n/CNTT/issues/385
Following discussion and approval during 09/10 chapter meeting
Modifications of Table4-24 "Mapping of NFVI capabilities to instance types"
- For Compute Intensive instance type: NUMA support -> Yes, Huge Pages ->Yes
- Rows "i.nfvi.mon.cap.005" and "i.nfvi.mon.cap.006" deleted, as they aren't capabilities but measurements

In addition, on Table 4-24, 
- deletion of rows: "i.nfvi.sec.cap.001",  "i.nfvi.sec.cap.002",  "i.nfvi.sec.cap.003",  "i.nfvi.sec.cap.004" as they aren't described in chapter 4, but have been relocated to chapter 7 Security
- For Basic instance type: "i.nfvi.mon.cap.007" -> No

### CODE REVIEW

**Comment by rabi-abdel** [comment]
@karinesevilla let me know when you are happy with this PR and I will put it in 48 hours cool off period. @markshostak can you also please approve it.

**Comment by karinesevilla** [comment]
@rabi-abdel OK for me

File: doc/ref_model/chapters/chapter04.md [hunk 1]
```diff
@@ -532,18 +532,13 @@ n100, n200, n300, n400, n500, n600 | N | Y | N
...
 | `i.nfvi.mon.cap.001`<br />(Host CPU usage) | Yes | Yes | Yes | Internal monitoring capabilities as per [**Table 4-8**](#Table4-8) |
 | `i.nfvi.mon.cap.002`<br />(Virtual compute CPU usage) | Yes | Yes | Yes | |
 | `i.nfvi.mon.cap.003`<br />(Host CPU utilization) | Yes | Yes | Yes | |
 | `i.nfvi.mon.cap.004`<br />(Virtual compute CPU utilization) | Yes | Yes | Yes | |
-| `i.nfvi.mon.cap.005`<br />(External storage IOPS) | Yes | No | Yes | |
-| `i.nfvi.mon.cap.006`<br />(External storage throughput) | Yes | No | Yes | |
-| `i.nfvi.mon.cap.007`<br />(External storage capacity) | Yes | No | Yes | |
-| `i.nfvi.sec.cap.001`<br />(VNFC/VNFC memory isolation) | Yes | Yes | Yes | Internal security capabilities as per [**Table 7-1**](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter07.md#Table7-1) |
-| `i.nfvi.sec.cap.002`<br />(VNFC -> Host) | No | No | No | |
-| `i.nfvi.sec.cap.003`<br />(Host -> VNFC) | Yes | Yes | Yes | |
```
Comment (markshostak) [inline doc/ref_model/chapters/chapter04.md]: Has there been any follow-up WRT the points @kedmison raised on Issue #385? I think they have to be addressed before these lines can be deleted. I'm good w/ the rest of the edit.

**Comment by rabiabdel** [comment]
@markshostak @karinesevilla 
this PR require resolving
<<<<<<< karinesevilla-patch-1
| `i.nfvi.per.cap.001`<br />(Huge page support) | No | Yes | Yes | Internal performance capabilities as per [**Table 4-7**](#Table4-7) |
| `i.nfvi.mon.cap.001`<br />(Host CPU usage) | Yes | Yes | Yes | Internal monitoring capabilities as per [**Table 4-8**](#Table4-8) |
=======
| `i.nfvi.per.cap.001`<br />(Huge page support) | No | Yes | No | Internal performance capabilities as per [**Table 4-7**](#Table4-7) |
| `i.nfvi.mon.cap.001`<br />(Host CPU usage) | Yes | Yes | Yes | Internal PM capabilities as per [**Table 4-8**](#Table4-8) |
>>>>>>> master

which line/combination you would like to keep?

**Comment by rabiabdel** [comment]
@markshostak New Process kicked in :) so we need 4 approvals in total before getting it merged. @markshostak can you please propose one more approval from operators and 2 more approvals from vendors? i will leave it to you to decide who those should be

**Comment by markshostak** [comment]
@karinesevilla Can you update the PR with the new Issue numbers please? Thanks, -Mark

**Comment by rabiabdel** [comment]
trying to add @trevgc test

**Comment by karinesevilla** [comment]
@rabi-abdel keep the following
| i.nfvi.per.cap.001
(Huge page support) | No | Yes | Yes | Internal performance capabilities as per Table 4-7 |
| i.nfvi.mon.cap.001
(Host CPU usage) | Yes | Yes | Yes | Internal monitoring capabilities as per Table 4-8

**Comment by karinesevilla** [comment]
@markshostak 
Following the discussion we had during RM Chapter 4 meeting (10/16), 3 issues have been created:
- RM Chap07 issue#431
- RA1 Chap06 issue#432
- RA2 Chap06 issue#433

**Comment by kedmison** [comment]
I resolved the merge conflict.

**Pull request merged**